src/stdlib_linalg.fypp

```diff
@@ -63,6 +65,22 @@ module stdlib_linalg
     #:endfor
   end interface
 
+
+  ! Outer product (of two vectors)
+  interface outer_product
+    !! version: experimental
+    !!
+    !! Computes outer product of two vectors, returning a matrix
+    !! ([Specification](../page/specs/stdlib_linalg.html#description_3))
+    #:for k1, t1 in RCI_KINDS_TYPES
+     module function outer_product_${t1[0]}$${k1}$(u,v) result(res)
+       ${t1}$, intent(in) :: u(:), v(:)
+       ${t1}$ :: res(size(u),size(v))
+       integer :: col
```

Comment:
I think it's best to leave `col` in the submodule part with the implementation.

Reply:
Definitely, I just copy and pasted indiscriminately from the submodule.